src/lib/dfp/Advert.spec.ts

```diff
@@ -157,3 +158,32 @@ describe('getAdSizeMapping', () => {
 		},
 	);
 });
+
+describe('findSmallestAdHeightForSlot', () => {
+	it.each([
+		['inline', 'desktop', 250],
+		['inline', 'phablet', 197],
+		['inline', 'tablet', 197],
+		['inline', 'mobile', 197],
+		['top-above-nav', 'desktop', 90],
+		['top-above-nav', 'tablet', 90],
+		['top-above-nav', 'phablet', 197],
+		['top-above-nav', 'mobile', 197],
+		['fronts-banner', 'mobile', null],
+		['fronts-banner', 'tablet', null],
+		['fronts-banner', 'desktop', 250],
+		['right', 'mobile', 250],
+		['right', 'tablet', 250],
+		['right', 'desktop', 250],
+	])(
+		'should find the smallest size for %s slot at %s breakpoint',
+		(slot, breakpoint, expected) => {
+			expect(
+				findSmallestAdHeightForSlot(
+					slot as AdSizesType.SlotName,
+					breakpoint as Breakpoint,
```

Comment:
Take or leave this one, but you could probably avoid asserting these here types here by asserting `as const` on the array of test cases.